client/src/hooks/networks/index.ts

```diff
@@ -53,27 +47,134 @@ type NetworkProperties = {
   projects: ProjectProperties[];
 };
 
-type PointFeatureWithNetworkProperties = PointFeature<NetworkProperties>;
+export type PointFeatureWithNetworkProperties = PointFeature<NetworkProperties>;
 
-export type NetworkMapResponse = {
-  features: PointFeatureWithNetworkProperties[];
+export type NetworkResponse = {
+  networks: OrganizationListResponseDataItem[] | ProjectListResponseDataItem[];
   isFetching: boolean;
   isFetched: boolean;
   isPlaceholderData: boolean;
   isError: boolean;
 };
 
-export const useMapNetworks: () => NetworkMapResponse = () => {
+const useGetOrganizationsWithFilters = (filters: Filters | undefined) => {
```

Comment:
I'm confused why this function is called `useGetOrganizationsWithFilters` but then checks the `type` later. Could you rename it?

For example, if it's used to get the related organisations, maybe `useGetRelatedOrganizationsWithFilters` is more appropriate?

Reply:
it was useGetNetworksWithFilters actually. Thanks